packages/block-library/src/navigation/edit/index.js

```diff
@@ -961,19 +941,7 @@ function Navigation( {
 								/>
 							) }
 						{ isOffCanvasNavigationEditorEnabled && (
-							<Button
-								variant="link"
-								className="wp-block-navigation-manage-menus-button"
-								disabled={
-									! hasManagePermissions ||
-									! hasResolvedNavigationMenus
-								}
-								href={ addQueryArgs( 'edit.php', {
-									post_type: 'wp_navigation',
-								} ) }
-							>
-								{ __( 'Manage menus' ) }
-							</Button>
+							<ManageMenusButton className="wp-block-navigation-manage-menus-button" />
```

Comment:
How come we need a specific classname here? Curious...

Reply:
Its for when the button is used in the advanced section, when the edit button is turned on. I tried removing that need too, but it got messy. I think ideally this button would live in the navigation menu selector anyway....